eispy/calibration/pixel_calibration.py

```diff
@@ -224,10 +224,10 @@ def _construct_hot_warm_pix_url(date, pix_type, detector, top_bot, left_right):
     url += pix_type + '/'
     url += date.strftime("%Y-%m-%d") + '/'
     url += 'coords_' + detector + '_' + left_right + '_'
```

Comment:
This is to do with writing the file (level 1) and not being in standard... I'll have to look in detail.